### ISSUE TITLE
VIA is not launchable in macOS Catalina

### ISSUE THREAD

**Issue by garrettmurray** [the-via/releases#1]
Running version 10.15.2. Confirmed `spctl` is disabled and Security setting set to _"Allow apps downloaded from: Anywhere"_ just in case the app isn't signed.

Finder error is _"The application 'VIA' can’t be opened."_

Some logging I see in console:

```
Checking whether application is managed at file:///Applications/VIA.app/org.via.configurator
```

```
Received xpc stream event (distributed notification matching) with name: com.apple.LaunchServices.applicationRegistered user info: {
    CFBundleDisplayName = VIA;
    bundleIDs =     (
        "org.via.configurator"
    );
    isPlaceholder = 0;
}
```

```
LAUNCHING:0x0-0x5d15d1 VIA foreground=1 bringForward=1 seed=1738 userActivityCount=0
```

```
spawn_via_launchd() failed, errno=111 label=[0x0-0x5d15d1].org.via.configurator path=/Applications/VIA.app/Contents/MacOS/VIA flags=1
```

**Comment by garrettmurray** [comment]
Apologies if this isn't the best place to file this issue—please feel free to point me in the right spot for this if not. It wasn't entirely clear to me where VIA is living now.

**Comment by olivia** [comment]
I believe this is likely a lack of code signing/notarization issue. I’ve been able to reproduce this on Catalina when trying to run the VIA app from the .zip file. You should be able to open the .dmg file instead and drag it over to Applications and do the right click -> open dance until OSX let’s you open it. 

This is the appropriate place for filing issues, thank your for the report!

**Comment by garrettmurray** [comment]
Yep, the DMG totally works—I apologize for not trying that first. I made the assumption that the DMG was the same exact build of VIA as contained in the zip. Got it working now, thanks so much for your quick response!

(Out of curiosity... what _is_ the difference between the builds that makes the DMG version work?)

**Comment by olivia** [comment]
The `zip` and `dmg` versions of VIA *should* be identical however it seems like there is a bug with the current packager that is creating this error when zipping the application: https://github.com/electron-userland/electron-builder/issues/4299

**Comment by garrettmurray** [comment]
Awesome, thanks for the info. Feel free to leave this open for an eventual fix to pin to it or close it, either way you solved this for me in the short-term!

**Comment by falzm** [comment]
I confirm version 1.2.5 works fine on macOS 10.15.3 👍
